aiohttp/web_log.py

```diff
@@ -189,6 +189,8 @@ def _format_line(
         return [(key, method(request, response, time)) for key, method in self._methods]
 
     def log(self, request: BaseRequest, response: StreamResponse, time: float) -> None:
+        if not self.logger.isEnabledFor(logging.INFO):
```

Comment:
I think this deserves a code comment with motivation, at least...

Reply:
added a comment